pkg/ceph/rgw/admin.go

```diff
@@ -49,7 +49,7 @@ func RunAdminCommand(c *Context, command, subcommand string, args ...string) (st
 	options = append(options, args...)
 
 	// start the rgw admin command
-	output, err := c.context.Executor.ExecuteCommandWithCombinedOutput(false, "", "radosgw-admin", options...)
+	output, err := c.context.Executor.ExecuteCommandWithOutput(false, "", "radosgw-admin", options...)
```

Comment:
what impact does losing stderr have on other usages of `RunAdminCommand`, such as list buckets, delete bucket, create user?

do we still get a useful error message if those were to fail?

Reply:
i'm wondering if we should wait on this change until there is a fix upstream for the output file. For now there are two approaches. 
- Use the combined output. If debug logging is enabled it will fail. But this will only be an issue for the operator and api pods. But those pods probably aren't where the ceph debug logging would be enabled anyway.  We wait for a fix upstream to get an out-file option. 
- Ignore the stderr. On failure we could retry with stderr captured so we could have the log. Seems hackish to retry just for the stderr.

I'm leaning towards the first option of doing nothing since it's not a scenario to enable the debug logging in the operator/api pods.